sdk/driver/integration/suite_test.go

```diff
@@ -114,8 +115,10 @@ func (s *suite) doTestNativeParse(t *testing.T, filename string) {
 	r.Nil(err)
 
 	expected := getAST(r, filename)
+	resstr, err := common.NativeParseResponseToString(res)
```

Comment:
`resstr` doesn't sound like the best name for a variable.

Reply:
Why? Is a response as string.

Reply:
`resstr` out of context is rather hard to understand.
If String method is used then using `res.String()` in the assertion could be enough.

Reply:
Well, it's just after res that is the response :) but I'll change it to just `str` that gives context enough, I think.

The assertion can't use res.String() because that code is in the integration tests suite that checks against the generated files.

Reply:
Hmmm, I fail to see why you cannot use/implement/move a `String` method and instead a `NativeParseResponseToString` is needed.

Reply:
Because gRPC (or Proteus) adds a default implementation of String to NativeParseReponse (in `generated.pb.go` and I didn't want to change that one, so I reused this `NativeParseResponseToString` that was already defined in the integration test suite since the rewrite.

Let me check if there is some way to provide a custom implementation for `String` in that object and it doesn't break anything.

Reply:
I see, ok.

Reply:
I made it work with `.String()`, the generated protobuf files change so I'll update dependent projects (everything seems to work but just for peace of mind...).